.github/workflows/release.yml

```diff
@@ -0,0 +1,33 @@
+name: Publish a release
+
+on:
+  push:
+    tags:
+      - 'v**'
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Get version
+        id: package-version
+        uses: martinbeentjes/npm-get-version-action@master
+      
+      - name: Create a release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: v${{ steps.package-version.outputs.current-version}}
+          release_name: "V${{ steps.package-version.outputs.current-version}}"
+      
+      - name: Publish to NPM
+        run: npm publish --access public
+        env:
+          VERSION: ${{ steps.package-version.outputs.current-version}}
```

Comment:
How about dropping the "v" at the start of the tag ? It would allow us to use `$GITHUB_REF` directly, which is the tag on `release` event.

Reply:
Yup, in this case use `${GITHUB_REF#/refs/tags/}` to truncate `/refs/tags/` from the variables I guess. But it would be usefull for the release event only.

Reply:
I choose not droping the `v`since previous tag embed `v` we can discuss about that later ?